app-i18n/man-pages-zh_CN/man-pages-zh_CN-1.6.3.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A somewhat comprehensive collection of Chinese Linux man pages"
+HOMEPAGE="https://github.com/man-pages-zh/manpages-zh"
+MY_PN="manpages-zh"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="https://github.com/${PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="FDL-1.2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE=""
+
+RDEPEND="virtual/man"
+
+src_prepare() {
+    # remove man pages that are provided by other packages.
+    # - sys-apps/shadow +nls
+    rm -f src/man1/{chfn,chsh,newgrp,su,passwd,groups}.1 || die
+    rm -f src/man8/{vigr,vipw}.8 || die
+
+    eapply_user
+}
+
+src_configure() {
+    :
+}
+
+src_compile() {
+    :
+}
+
+src_install() {
+    for x in src/man?/*.[1-9]*; do
+        doman -i18n=zh_CN $x
+    done
+    dodoc README.md* AUTHORS* ChangeLog* NEWS*
```

Comment:
Why all the asterisks? They don't seem to catch anything.

Reply:
Fixed everything in last commit, could you please have a review?